common/logger/src/libra_logger.rs

```diff
@@ -304,6 +304,38 @@ impl Writer for StderrWriter {
     }
 }
 
+/// A struct for writing logs to a file
+pub struct FileWriter {
```

Comment:
What's the rationale for writing to a file and not just to stderr?

Reply:
ahh, I put a comment in the main node code, but I'm trying to eliminate more documentation and just making things easy. when we run the test mode, it prints out all the useful information to get connected to it... I don't think most users are aware that what is visible on a terminal can be either err or out. so ... let's just put it into a file and reduce educational burden.

Reply:
fair enough